knowage-vue/src/modules/kpi/cron/Cron.vue

```diff
@@ -0,0 +1,394 @@
+<template>
```

Comment:
Is this Cron.vue file a component that could be shared between different functionalities?
In that case should be inside the src/components folder

Reply:
We are changing it on our master, i will create another pull request soon

Reply:
Perfect thank you